website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -110,138 +115,159 @@ Here is the current architecture of our application:
 
 ```
-$ exo setup
+exo run
 ```
 
-We see how it uses Node's package management system (NPM)
-to download and install
-the external [ExpressJS](http://expressjs.com) and [Pug](http://jade-lang.com/) (formerly Jade) modules for us,
-so that the service is ready to run.
 The output should look something like:
 
 ```
-Setting up Todo application 0.0.1
-
-  exo-setup  starting setup of 'html-server'
-html-server  /Users/kevin/exosphere-sdk/tutorial/part_2/code_02/todo-app/html-server
-├── express@4.13.4
-└── jade@1.11.0
-html-server  PROCESS ENDED
-html-server  EXIT CODE: 0
-  exo-setup  setup of 'html-server' finished
-  exo-setup  setup complete
-```
-
-
-## Booting up the application
-
-To test that everything works, let's check that the application boots up:
-
-```
-$ exo run
+>>> docker-compose --file run_development.yml up --build
+Creating network "todoapp_default" with the default driver
+Pulling exocom0.26.1 (originate/exocom:0.26.1)...
+0.26.1: Pulling from originate/exocom
+add3ddb21ede: Pull complete
+7b0e69015745: Pull complete
+Digest: sha256:56e176f25e21732ccad671375fba8177a04a3fc7ea8bc1decf9e715c6af961c6
+Status: Downloaded newer image for originate/exocom:0.26.1
+Building web-service
+Step 1/6 : FROM node
+latest: Pulling from library/node
+85b1f47fba49: Pull complete
+ba6bd283713a: Pull complete
+817c8cd48a09: Pull complete
+47cc0ed96dc3: Pull complete
+8888adcbd08b: Pull complete
+6f2de60646b9: Pull complete
+9dd205971dc0: Pull complete
+5859715a4691: Pull complete
+Digest: sha256:7c9099e0f68242387d7755eaa54c287e16cedd3cca423444ca773794f5f1e423
+Status: Downloaded newer image for node:latest
+ ---> c1d02ac1d9b4
+Step 2/6 : ENV NODE_ENV "development"
+ ---> Running in 61ad323a3101
+ ---> 0d4ade2ed6a6
+Removing intermediate container 61ad323a3101
+Step 3/6 : COPY package.json .
+ ---> 382f908e332a
+Step 4/6 : COPY yarn.lock .
+ ---> 2b2387775f86
+Step 5/6 : RUN yarn
+ ---> Running in e91437d97867
+yarn install v1.3.2
+[1/4] Resolving packages...
+[2/4] Fetching packages...
+info fsevents@1.1.2: The platform "linux" is incompatible with this module.
+info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
+[3/4] Linking dependencies...
+[4/4] Building fresh packages...
+Done in 10.40s.
+ ---> 714bc67a7dc4
+Removing intermediate container e91437d97867
+Step 6/6 : COPY . .
+ ---> 5a6ff5c8cff8
+Successfully built 5a6ff5c8cff8
+Successfully tagged todoapp_web-service:latest
+Creating exocom0.26.1 ...
+Creating exocom0.26.1 ... done
+Creating web-service ...
+Creating web-service ... done
+Attaching to exocom0.26.1, web-service
+exocom0.26.1    | ExoCom online at port 80
+exocom0.26.1    | 'web-service' registered
+web-service     | ExoRelay for 'web-service' online at port 80
+web-service     | HTML server online at port 3000
+web-service     | HTML server is running
 ```
 
-You should see output like:
+The Exosphere framework itself is written as a bunch of loosely coupled
```

Comment:
This isn't correct anymore. Exosphere is now a single monolithic Go binary. Exocom is a different product, and the web service is part of the application.

Reply:
Were you implying that I should remove the reference to exocom from the list? I redid this but kept the text referring to exocom. If that isn't desired I will adjust.

Reply:
Let's focus this tutorial on Exosphere and leave Exocom and micro-services out as much as possible. Exosphere works equally well for monoliths. We can use Exocom in the example application, but don't have to go into details on how that application works.